0
50
100
150
200
250
300
350
400
450
500
550
600
650
700
750
800
Epoch 0 loss: 1878.042236
Traceback (most recent call last):
  File "/home/gpufs/users/students/iasd23/iasd23_ainane/master_iasd/data_challenge/train.py", line 80, in <module>
    torch.save(autoencoder,f'{model_base_name + str(epoch + resume_epoch + 1)}.pth')
                                                            ^^^^^^^^^^^^
NameError: name 'resume_epoch' is not defined
0
50
100
150
200
250
300
350
400
450
500
